wandb/sdk/internal/job_builder.py

```diff
@@ -259,18 +241,54 @@ def _build_image_job_source(
 
         raw_image_name = image_name
         if ":" in image_name:
-            raw_image_name, tag = image_name.split(":")
-            self._aliases += [tag]
+            tag = image_name.split(":")[-1]
+
+            # if tag looks properly formatted, assume its a tag
+            # regex: alphanumeric add "_" "-" "."
```

Comment:
```suggestion
            # regex: alphanumeric and "_" "-" "."
```

Reply:
This is all associated with the second ticket you added in here right?